providers/snowflake/view.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 The Terraformer Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package snowflake
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraform_utils"
+)
+
+type ViewGenerator struct {
+	SnowflakeService
+}
+
+func (g ViewGenerator) createResources(viewList []view) []terraform_utils.Resource {
+	var resources []terraform_utils.Resource
+	for _, view := range viewList {
+		resources = append(resources, terraform_utils.NewSimpleResource(
+			fmt.Sprintf("%s|%s|%s", view.DatabaseName.String, view.SchemaName.String, view.Name.String),
+			fmt.Sprintf("%s|%s|%s", view.DatabaseName.String, view.SchemaName.String, view.Name.String),
```

Comment:
shouldn't these | be _?

Reply:
only for the name and not the id?

Reply:
https://github.com/chanzuckerberg/terraform-provider-snowflake/blob/master/pkg/resources/view.go#L94

Reply:
yep, so the ID of the resource for sure needs to have the `|`.
but the name of the resource (resource snowflake_view <NAME>) should not. because terraformer will filter it out. I think it would be clearer if we did the _ or the __ like in the other resources.

Reply:
that way you end up with `resource snowflake_view database_name__schema_name__view_name{}` blocks in terraform

Reply:
that way you end up with `resource snowflake_view database_name__schema_name__view_name{}` blocks in terraform

instead of `database_nameschema_nameview_name` (since terraformer filters out the |)

Reply:
ah I see. sounds good!